### PR TITLE
GPC-187: ADR for SQS

### PR DESCRIPTION
**Pull request by ethanmills**
For discussion

### CODE REVIEW

File: doc/architecture/decisions/0016-push-directly-to-sqs.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+More broadly, the API surface of the system will be expanded to include the acquirer queue as well as the HTTP REST API.
+AWS services such as SQS and S3 may be considered for inclusion in the API surface exposed, whereas third party systems
+are more likely to need an adaptor. Authentication to AWS services can happen with ephemeral credentials created using IAM
+roles, but external services will need longer term credentials. It is also easier to monitor compliance with policies using
+built in AWS tooling. We should therefore consider anything external to AWS to outside of the "core" service and so
+requiring considerations.
+
+## Impact
+Exposing acquirer queues to the kotlin service will mean that queue clients need to be generated dynamically from data in
```
Comment (olevett) [inline doc/architecture/decisions/0016-push-directly-to-sqs.md:38]: With this, do we also implicitly take on creating queues as well as clients from our service?

**Comment by olevett** [comment]
I think I'm broadly happy with this as an approach and justification. I think as/when we've implemented this, it would be good to have a brief review and validate it is working as we think it should.

File: doc/architecture/decisions/0016-push-directly-to-sqs.md (hunk 1 above)
Comment (ethanmills) [reply doc/architecture/decisions/0016-push-directly-to-sqs.md:38]: Yes, I can make this explicit

**Comment by oskarwilliams378** [comment]
I agree with Olly, I think this approach makes sense with out current expectation of the system

**Comment by Mjwillis** [comment]
Yes sounds like a good starting point - suspect we'll come up with some interesting issue on route when we implement but I like to idea of taking away the complexity of setup on the acquirers side as much as we can.

**Review by Mjwillis — APPROVED** [review]
See comments

**Pull request merged**